File: pset6/mario_less.py
import cs50

def main():
    while True:
        height=cs50.get_int("Height: ")
        if height>=0 and height<=23:
            break
    for i in range(2,height+2):
        print(" "*(height+1-i),end="")
        print("#"*i)

if __name__=="__main__":
    main()

File: pset1/mario_less.c
#include<stdio.h>
#include<cs50.h>

int main(){
    int x;
    do{
        printf("Height: ");
        x=get_int();
    }while(x<0||x>23);
    for(int i=2;i<=x+1;i++){
        for(int j=0;j<x+1-i;j++) printf(" ");
        for(int j=0;j<i;j++) printf("#");
        printf("\n");
    }
}